turbopack/crates/turbopack-core/src/resolve/mod.rs

```diff
@@ -2865,7 +2865,10 @@ pub enum ModulePart {
     /// all exports are unused.
     Evaluation,
     /// Represents an export of a module.
-    Export(Vc<RcStr>),
+    ///
+    ///
+    /// bool value is true if it's proxied for `export * from './foo'`
+    Export(Vc<RcStr>, Vc<bool>),
```

Comment:
Not sure if you still need that, but here are few general hints:

* Vc<bool> is not useful here.
* You might want to change it to named fields to give to boolean a name

```suggestion
    Export(Vc<RcStr>, bool),
```